server/src/main/java/fr/uga/l3miage/tp1/exo3/models/BrandEntity.java

```diff
@@ -0,0 +1,26 @@
+package fr.uga.l3miage.tp1.exo3.models;
+
+import javax.persistence.*;
+
+@Table(name = "brand_miage")
+@Entity
+public class BrandEntity {
+
+    // pas de PK?
+    // Mais une FK
+    @Id
+    @Column(name = "company_name")
+    private String company_name;
+
+    @Column(name = "siretNumber", length = 11)
```

Comment:
attention en base ce n'est que du `snake_case`
```suggestion
    @Column(name = "siret_number", length = 11)
```